production/helm/loki/Chart.yaml

```diff
@@ -4,7 +4,7 @@ name: loki
 description: Helm chart for Grafana Loki in simple, scalable mode
 type: application
 appVersion: 2.7.0
-version: 4.1.0
+version: 4.1.1
```

Comment:
I think we can just to a minor version for this. New functionality, so not really a patch IMO